intl/msg_hash_us.h

```diff
@@ -1480,6 +1524,18 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_AUDIO_RESAMPLER_DRIVER,
    "Audio resampler driver to use."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_AUDIO_RESAMPLER_DRIVER_SINC,
+   "Windowed SINC implementation."
```

Comment:
I recall reading this as "Sinc" and not "SINC" elsewhere in Crowdin (perhaps in a core implementation and not the main app). Is this correct?